ffxivdpscalc/classimpl/sch.py

```diff
@@ -0,0 +1,47 @@
+import math
+
+from ffxivdpscalc.calc import CharacterStats
+
+
+class SchCharacterStats(CharacterStats):
```

Comment:
this inheritance seems not quite right. 

i think this class should be SchPPS inheriting from HealerPPS inheriting from CasterPPS inheriting from PPS (to allow future PPS sims to share related calculations, etc). the base pps class should have a calculate_pps function that takes char stats as input, and each job's pps class should override that function with the actual calculation.

Reply:
PTAL